### DISCUSSION TITLE
tikv as kv storage error

### DISCUSSION THREAD

**Discussion by suzhouclark** [juicedata/juicefs#693] [Q&A] [answered]
there is error when I want to use tikv as kv storage 
====================================
tikv env
tiup playground nightly --mode tikv-slim
![image](https://user-images.githubusercontent.com/3823093/127735936-060f3c2e-3ce6-4f96-8819-20cadda58850.png)
====================================
juicefs format  error
![image](https://user-images.githubusercontent.com/3823093/127735975-c69568c0-6526-4d5e-9f64-11258e5678ce.png)

**Comment by davies** [comment, marked as answer]
What's the version of JuiceFS? 

TiKV support is not built by default, please build it with 
```
make juicefs.tikv
```
